be/src/storage/olap_common.h

```diff
@@ -376,7 +376,17 @@ struct Version {
 
     bool contains(const Version& other) const { return first <= other.first && second >= other.second; }
 
-    bool operator<(const Version& rhs) const { return second < rhs.second; }
+    bool operator<(const Version& rhs) const {
```

Comment:
could you add some comments about in which scene will exists overlap version in rs_version_map